example_values/f5-bigip-ingress/name-based-specific.yaml

```diff
@@ -0,0 +1,58 @@
+# Values template for f5-bigip-ingress
+# Suggested use is to copy, specify values, and pass with `-f` to helm install
+ingress:
+  namespace: ~
```

Comment:
If these are example values why are you using a nil for namespace? Couldn't we just have like a default or a f5 namespace here?

Reply:
The null value is being used to indicate that this is something you *should* specify - i.e we expect you to have an intention for that value. In some cases (bigip url) not having the value throws an error. In the case of namespace, if you don't specify one, k8s uses default (and adds it to the manifest).

Reply:
Cool sounds good to me then. You should add that comment that you just told me somewhere, because that could be very helpful for users.

Reply:
I'll add it to the docs story referenced below.

Reply:
Awesome.